tests/foreman/cli/test_ansible.py

```diff
@@ -931,37 +937,43 @@ def test_positive_ansible_provisioning_callback(
         :expectedresults:
             1. All hosts managed by Satellite are added to Satellite inventory.
             2. Starting ansible-callback systemd service, starts a job_template execution in AAP
+
+        :verifies: SAT-30761
         """
         host_mac_addr = provisioning_host.provisioning_nic_mac_addr
         sat = module_provisioning_sat.sat
-        aap_fqdn = settings.AAP_INTEGRATION.AAP_FQDN
+        aap_fqdn = settings.AAP_INTEGRATION.get(
+            'AAP25_FQDN' if aap_version == '2.5' else 'AAP23_FQDN'
```

Comment:
Where do we set `AAP25_FQDN`/`AAP23_FQDN`?

Reply:
see satellite-jenkins MR#1638